package.json

```
@@ -71,7 +71,8 @@
     "release-it": "^17.3.0",
     "typescript": "^5.4.5",
     "unbuild": "^2.0.0",
-    "valibot": "^0.30.0",
+    "valibot30": "npm:valibot@0.30.0",
+    "valibot31": "npm:valibot@^0.31.1",
```

Shouldn't it be better for typings to keep the latest version as `valibot`?

```suggestion
    "valibot": "^0.31.1",
```

---

I believe there is no difference. I just followed a StackOverflow discussion to install two different versions of the same package. If we change the package name, we also need to update the import statement.

---

Yeah in the `Form.vue` component we could import from `valibot30` and `valibot` (instead of `valibot31`).

---

Done